src/lib.rs

```diff
@@ -152,12 +168,10 @@ impl Wiki {
     }
 
     // Upvote or download an article
+    #[private]
```

Comment:
The same here. Even as contract owner, we are not interested in calling this endpoint.